forch/http_server.py

```diff
@@ -45,6 +45,8 @@ def do_GET(self):
                 opts[pair[0]] = pair[1]
             message = str(self._context.get_data(self.headers.get('Host'), parsed.path[1:], opts))
             self.send_response(http.HTTPStatus.OK)
+            if self._context.content_type:
+                self.send_header('Content-type', self._context.content_type)
```

Comment:
Does this apply to everything, or only varz requests?

Reply:
Yup. Not for the normal forch page

Reply:
So how does it not get applied to everything?  From a quick read, it seems that it will get applied to everything that goes through the proxy.  How is it limited to varz pages?